setup.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+# Package meta-data
+NAME = "hepdata-pyhf-extractor"
+INFO = "Python package to summarize pyhf submissions"
+URL = "https://github.com/HEPData/hepdata-pyhf-extractor"
+REQUIRES_PYTHON = ">=3.6, <4"
+AUTHORS = "HEPData and pyhf teams"
+VERSION = open("VERSION", "r").read().strip()
+
+
+# Installation requirements
+INSTALLATION_REQS = []
+
+# Development requirements
+DEVELOPMENT_REQS = [
+    "black>=20.8b1",
+    "coverage>=5.0.4",
+    "pre-commit>=2.4.0",
+    "pytest>=6.0.2",
+    "pytest-cov>=2.10.0",
+]
```

Comment:
```suggestion
    "bump2version",
]
```

Following up from comment above in `Makefile`.

Reply:
`bump2version` added.

Please review, I am not that familiar with this tool.